Source/FakeItEasy.Tests/Core/DynamicContainerTests.cs

```diff
@@ -116,9 +116,9 @@ private DynamicContainer CreateContainer()
 
         public class OptionsBuilderForTypeWithDummyFactory : IFakeOptionsBuilder
         {
-            public int Priority
+            public Priority Priority
             {
-                get { return 0; }
+                get { return new Priority(0); }

```

Comment:
Perhaps we should add a `Priority.Default` property instead of writing `new Priority(0)` every time? And we could also have an internal `Priority.Internal` property for `new Priority(-1)`. Not really important, but it looks more elegant IMO...